web/yo/app/scripts/directives/curationQueue.js

```diff
@@ -584,6 +596,21 @@ angular.module('oncokbApp')
                         return 0;
                     }
                 }
+                $scope.getQueuesByGene = function(hugoSymbol) {
+                    return $rootScope.firebaseQueues[hugoSymbol] ? angular.copy($rootScope.firebaseQueues[hugoSymbol].queue) : [];
+                };
+                $scope.updateQueueInDB = function (hugoSymbol, updatedQueues) {
+                    var defer = $q.defer();
+                    firebase.database().ref('Queues/' + hugoSymbol).set({
+                        queue: updatedQueues
+                    }).then(function(result) {
+                        defer.resolve('success');
+                    }).catch(function(error) {
+                        dialogs.error('Error', 'Fail to save changes to database. Please contact developer!');
```

Comment:
This error message should be saved somewhere in the global vars, then it can be used everywhere.

Reply:
I see your point to reuse redundant message variable. But right now, this message is unique. I will put it in a global varible when necessary.